### DISCUSSION TITLE
Proxy / Firewall

### DISCUSSION THREAD

**Discussion by thomasLeclaire** [envkey/envkey#85] [Q&A] [answered]
Hello! 
what are the host / port which have to be allowed on some local proxy/firewall to allow the Windows UI to work? 

I saw there are at least https://envkey-releases.s3.amazonaws.com/latest/envkeysource-version.txt but sounds not enough. 

thx!

**Comment by danenania** [comment, marked as answer]
Hi @thomasLeclaire,

- https://api-v2.envkey.com (api)
- https://api-v2-2.envkey.com (failover for envkey-source requests)
- https://envkey-releases.s3.amazonaws.com (multiple paths are needed from this bucket to enable upgrades--best if you can enable the full subdomain I think)

I think enabling those should do it.
